DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -129,6 +132,14 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
         super.init()
 
         try? configureUpdater()
+
+#if DEBUG
+        updater?.automaticallyChecksForUpdates = false
+        updater?.automaticallyDownloadsUpdates = false
+        updater?.updateCheckInterval = 0
+#else
+        updater?.checkForUpdatesInBackground()
```

Comment:
During initialization, only triggers a "background" update. This respects the phased update group handling.